generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
1. The hugepage size is 0 if you have not set 'expected_hugepage_size'.
2. if you have not set 'expected_hugepage_size', of course the code won't know what's the hugepage size you want.
so it's a must param in the hugepage case configurations, else it's 0 KB as default.
3. could you check your debug.log if there is a default value for 'expected_hugepage_size' and what's the value of 'setup_hugepages' ?

Thanks.